### PR TITLE
More tests for Pkg

### PR DESCRIPTION
**Pull request by kshyatt**
Few missed branches and methods.

### CODE REVIEW

File: test/pkg.jl [hunk 1]
```diff
@@ -75,6 +80,10 @@ temp_pkg_dir() do
     @test isempty(Pkg.installed())
     @test !isempty(Pkg.available("Example"))
     @test !in("Example", keys(Pkg.installed()))
+    Pkg.rm("Example") #second rm to make sure things are ok
```
Comment (tkelman) [inline test/pkg.jl]: what do you mean by "things are ok"?

Comment (kshyatt) [inline test/pkg.jl]: Testing [this](https://codecov.io/github/kshyatt/julia/base/pkg/entry.jl?ref=5e01b1ab5bfafe8e84c2569b19d43952e689dcda#l-82) (make sure nothing explodes horribly)

**Comment by wildart** [comment]
:+1:

File: test/pkg.jl (hunk 1 above)
Comment (tkelman) [inline test/pkg.jl]: could have changed the comment to something like "make sure calling Pkg.rm twice works" but whatever

**Pull request merged**